axelrod/strategies/_filters.py

```diff
@@ -0,0 +1,179 @@
+from distutils.util import strtobool
+from collections import namedtuple
+import operator
+
+
+def passes_boolean_filter(strategy, classifier_key, value):
+    """
+    Tests whether a given strategy passes a boolean equality filter for a
+    given key in its classifier dict.
+
+    Parameters
+    ----------
+        strategy : a descendant class of axelrod.Player
+        classifier_key: string
+            Defining which entry from the strategy's classifier dict is to be
+            tested (e.g. 'stochastic' or 'inspects_source').
+        value: string or boolean
```

Comment:
I've thought on this one a little further and changed my mind. I agree that it would be cleaner to do the string conversion in the API. I'll sort that out.